doc/Command-Reference.md

```diff
@@ -4116,6 +4116,36 @@ This command is used to enable or disable low-power mode for an SFP transceiver
   Disabling low-power mode for port Ethernet0...  OK
   ```
 
+**config interface transceiver frequency**
+
+This command is used to set the coherent module laser frequency
+
+- Usage:
+  ```
+  config interface transceiver frequency <interface_name> <frequency_value>
+  ```
+
+- Examples:
+  ```
+  user@sonic:~# config interface transceiver frequency Ethernet0 196100
+  Setting laser frequency to 196100 GHz on port Ethernet0
+  ```
+
+**config interface transceiver tx_power**
+
+This command is used to set the coherent module tx power
+
+- Usage:
+  ```
+  config interface transceiver tx_power <interface_name> <tx_power_value>
```

Comment:
is it tx-power or tx_power in the code? I think its tx-power. Please check

Reply:
it is tx_power